docs/install.md

```diff
@@ -18,7 +18,7 @@
 - CUDA 9.2+
 - GCC 5+
 - PyTorch3D 0.4+
-- [MMCV](https://github.com/open-mmlab/mmcv) (Please install mmcv-full>=1.3.17,<1.6.0 for GPU)
+- [MMCV](https://github.com/open-mmlab/mmcv) (Please install mmcv-full>=1.3.17,<1.8.0 for GPU)
```

Comment:
I successfully install MMHuman3D with the MMCV 1.7.1. but it requires us to choose the appropriate version of MMDet, MMTrack, MMPose, and MMCls. 

Here is a successful case:
MMCV -> 1.7.1
MMDet -> 2.28.2
MMTrack -> 0.14.0
MMCls ->  0.25.0.

If we only change here, users will not be able to install MMHuman3D due to the compatibility of MMDet, MMTrack, and MMCls. So I think the suggested install command should stay the same or we change the installation command of other dependent libraries at the same time?

Reply:
Let's stay on low version in the official install.md, but allow higher mmcv to be used in the __init__.py.